dask_snowflake/core.py

```diff
@@ -44,6 +44,7 @@ def write_snowflake(
         write_pandas(
             conn=conn,
             df=df,
+            schema=schema,
             # NOTE: since ensure_db_exists uses uppercase for the table name
```

Comment:
An aside: is this note still correct? I don't see any upper casing below, and as far as I can tell snowflake is case insensitive for table names

Reply:
My recollection is that the `to_sql` call here

https://github.com/coiled/dask-snowflake/blob/13fd136b5023478c30a83bbbd5bd777e4c76c4dd/dask_snowflake/core.py#L82-L90

is where the table name is automatically uppercased. Though it would be good to confirm this